spec/services/cycle_timetable_spec.rb

```diff
@@ -91,19 +105,19 @@
 
   describe '.find_down?' do
     it 'returns false before find closes' do
-      Timecop.travel(CycleTimetable.find_closes.beginning_of_day - 1.hour) do
+      Timecop.travel(CycleTimetable::CYCLE_DATES[2020][:find_closes].beginning_of_day - 1.hour) do
```

Comment:
nit: it'd be good to use the method interface rather than reaching down into the hash here. Maybe by adding an optional `year` param to `find_closes`?

Reply:
Yep good idea. Done

Reply:
https://github.com/DFE-Digital/apply-for-teacher-training/pull/5106/commits/eb018b3009f02d29bb6efc86dfd6994cd89b8e1c